vagrant/environment/puppet/modules/profile/files/mysql_grants.sql

```diff
@@ -1,6 +1,6 @@
-CREATE USER 'dba'@'%' IDENTIFIED BY 'qwerty';
-CREATE USER 'dba'@'localhost' IDENTIFIED BY 'qwerty';
-CREATE USER 'repl'@'%' IDENTIFIED BY 'qwerty';
+CREATE USER 'dba'@'%' IDENTIFIED WITH mysql_native_password BY 'qwerty';
```

Comment:
Not sure it's really needed. `mysql_native_password` is supposed to be the default as per my.cnf.